sound/soc/sof/ipc3-control.c

```diff
@@ -40,8 +42,11 @@ static int sof_ipc3_set_get_kcontrol_data(struct snd_sof_control *scontrol, bool
 	 * would always be > 0 in this case. For the others, just return the cached value if the
 	 * widget is not set up.
 	 */
+	if (lock)
```

Comment:
ths comment above is in the wrong place now

Reply:
OK